test/fixtures/output/iam/rbac/role-binding/list-invalid-role-error-type-1-cloud.golden

```diff
@@ -1 +1 @@
-Error: Invalid role name : InvalidOrgAdmin: 400 Bad Request
+Error: Invalid role name : InvalidOrgAdmin
```

Comment:
Unrelated, but can we remove this space?
```suggestion
Error: Invalid role name: InvalidOrgAdmin
```

Reply:
removed